### PR TITLE
Dummy PR to test workflow

### PR DESCRIPTION
**Pull request by aagarwal1012**
[Not meant for merging]

### CODE REVIEW

**Comment by awhitford** [comment]
Weird...  I was really only trying to change 1 thing -- and that worked, but the other inexplicably broke...  Let me try again.

**Comment by awhitford** [comment]
@aagarwal1012 I've tried 3 times and it always fails for me.  As an experiment, can you try replacing:

```yaml
      - run: |
          flutter pub get
          flutter build web --release
```

with

```yaml
      - run: flutter build web
```

And see if the secret works?  (Just like #228) Could the secret be limited to just you?

Note that the `build` step works fine; the failure is in the following line, which didn't change -- that makes it all the more puzzling.

**Comment by aagarwal1012** [comment]
> And see if the secret works? (Just like #228) Could the secret be limited to just you?

@awhitford, I think this seems to be the issue, you are creating a PR from your `fork`. That's why it is not able to access the secret. Instead, try creating a new branch in the main repository. You are a `collaborator`, so you have the right to create a new branch in the source repo. After this update, things should work fine!

**Comment by awhitford** [comment]
@aagarwal1012 PR #229 worked!  Thanks for the advice.

But shouldn't the GitHub Actions work for PRs from a person's fork? 🤔

**Comment by aagarwal1012** [comment]
@awhitford, github actions work, but source repo secrets are not accessible in case of forks otherwise it may cause security issues.

**Comment by SirusCodes** [comment]
Damn so many notifications in a day😂
